morango/models.py

```diff
@@ -293,6 +295,10 @@ class DatabaseMaxCounter(AbstractCounter):
     """
 
     partition = models.CharField(max_length=128, default="")
+    profile = models.CharField(max_length=40)
+
+    class Meta:
+        unique_together = ("instance_id", "partition")
```

Comment:
Does this need to also include "profile"? Since we might have an empty partition for each profile for the same instance_id? Or do we need "profile" on the model at all?